gcp/rakefiles/secrets.rb

```diff
@@ -77,45 +84,127 @@ def self.collect_secrets()
   def self.set_secrets(collected_secrets, exekube_cmd)
     return if collected_secrets.empty?
```

Comment:
Everything is wrapped in the collected_secrets.each block, so I think this line can be removed.

Reply:
Done.